src/vulkan-renderer/application.cpp

```diff
@@ -1,5 +1,7 @@
 #include "inexor/vulkan-renderer/application.hpp"
 
+#include "GLFW/glfw3.h"
+#include "inexor/vulkan-renderer/camera.hpp"
```

Comment:
Sort pls and use `<>` for third party includes.

Reply:
This is a rule that needs to be added to https://inexor-vulkan-renderer.readthedocs.io/en/latest/development/engine-design/main.html#coding-style by the way